test/e2e/traceflow_test.go

```diff
@@ -1159,6 +1159,17 @@ func testTraceflowInterNode(t *testing.T, data *TestData) {
 	agnhostIP, err := data.podWaitForIPs(defaultTimeout, agnhostPodName, data.testNamespace)
 	require.NoError(t, err)
 
+	node2TunnelDstIPv4, node2TunnelDstIPv6 := "", ""
+	encapMode, err := data.GetEncapMode()
+	require.NoError(t, err)
+
+	switch encapMode {
+	case config.TrafficEncapModeEncap:
+		node2TunnelDstIPv4 = clusterInfo.nodes[nodeIdx1].ipv4Addr
+		node2TunnelDstIPv6 = clusterInfo.nodes[nodeIdx1].ipv6Addr
+		// TODO: Add case for Hybrid mode when Nodes are in different subnet.
```

Comment:
I think the test in hybrid mode would fail if the nodes are not in the same subnet as the expectation is empty but the actual observation is not?

Reply:
Yes